server/src/domain/domain.constant.ts

```diff
@@ -67,33 +67,35 @@ const validMimeTypes = [
   'image/x-sony-arw',
   'image/x-sony-sr2',
   'image/x-sony-srf',
-  'image/dng',
+
+  'image/3fr',
```

Comment:
Ah, to clarify: Could we merge all of these into one contiguous list and then sort alphabetically?